eclipse-cdt-cloud/vscode-memory-inspector
# Introduce 'Endianness' in UI and honor it during group rendering

#### What it does

- Move Endianness interface into common code
- Add setting for 'Endianness' in package.json
- Add option for 'Endianness' in options panel
- Assume DAP as big endian and revert byte-order for little endian
- Set 'Little Endian' as default as it may be more common

![endianness](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/assets/19170971/a91f5518-abcc-4ec3-9cc2-bd4ab6e36b92)

Closes #88

#### How to test
- Open a Memory Inspector
- Test the default preference in the Settings
- Test the individual settings in the memory inspector options
- See how the byte order changes within a **group**, not only a word

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
